res/values-ar/strings.xml

```diff
@@ -1354,9 +1354,9 @@ Commands:
     <string name="CLAN_BACKGROUND_COLOR">لون خلفية العشيرة</string>
     <string name="activate">تفعيل</string>
     <string name="clan_id">رقم هوية العشيرة</string>
-    <string name="language_change_warning">سيؤدي تغيير اللغة إلى إعادة تشغيل التطبيق.</string>
-    <string name="xmas_gift_boost_message">لقد تلقيت هدية تعزيز البلازما 3 مرات ليوم واحد!</string>
-    <string name="no_response">لا توجد استجابة من الخادم. يرجى التحقق من اتصال الإنترنت.</string>
+    <string name="language_change_warning">  تحذير بشأن تغير اللغه .</string>
+    <string name="xmas_gift_boost_message"> رسالة تعزيز هديه عيد ميلاد!</string>
+    <string name="no_response">لا يوجد رد/string>
```

Comment:
Lines 1357 and 1358: Why insert extra space?
Line 1359: You simply translated "no response", so it was wrong. Please return it as it was or try making a valid correction basing on the `res/values` (English file).

Reply:
Sorry, I didn't notice the distance